src/ralph/virtual/admin.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+from django.core.urlresolvers import reverse
+
+from ralph.admin import RalphAdmin, RalphTabularInline, register
+from ralph.data_center.models.networks import IPAddress
+from ralph.data_center.models.physical import DataCenterAsset
+from ralph.virtual.models import (
+    CloudFlavor,
+    CloudHost,
+    CloudProject,
+    CloudProvider,
+    VirtualServer
+)
+
+
+@register(VirtualServer)
+class VirtualServerAdmin(RalphAdmin):
+    pass
+
+
+class CloudHostTabularInline(RalphTabularInline):
+    can_delete = False
+    model = CloudHost
+    fk_name = 'parent'
+    fields = ['get_hostname', 'get_hypervisor', 'get_ipaddresses', 'created',
+              'tags', 'remarks']
+    readonly_fields = ['created', 'get_hostname', 'get_hypervisor',
+                       'get_ipaddresses', 'host_id']
+
+    def get_hostname(self, obj):
+        return '<a href="{}">{}</a>'.format(
+            reverse("admin:virtual_cloudhost_change", args=(obj.id,)),
+            obj.hostname
+        )
+    get_hostname.short_description = 'Hostname'
+    get_hostname.allow_tags = True
+
+    def get_hypervisor(self, obj):
+        if obj.hypervisor is None:
+            return "Not set"
```

Comment:
wrap it in lazy translation (`from django.utils.translation import ugettext_lazy as _`)